pkg/model/clusterconfigentity.go

```diff
@@ -28,7 +28,7 @@ type ClusterConfigurationEntity struct {
 	ClusterVersion int64            `db:"notNull"` // Cluster entity primary key
 	KymaVersion    string           `db:"notNull"`
 	KymaProfile    string           `db:""`
-	Components     []*keb.Component `db:"notNull,encrypt"`
```

Comment:
field has to be encrypted actually

Reply:
sure, already added a comment above to clean it :)